stats.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Automattic\VIP\Stats;
+
+// Limit tracking to production
+if ( false !== VIP_GO_ENV ) {
```

Comment:
I think we're using `VIP_GO_ENV` to distinguish staging from production and using `WPCOM_IS_VIP_ENV` for local vs VIP Go. I guess we should change the comment here: https://github.com/Automattic/vip-go-mu-plugins/blob/master/000-vip-init.php#L21